Updated integration studio developer guide (#945)
wso2/integration-studio: gh-docs/developer-guide/integration-studio-catalogue.md

![](https://i.imgur.com/5L9ogrJ.png)

> Click the “Load” button which is beside the Model URI field. Then click “Next”.

Reviewer: ```suggestion
Click the “Load” button that is beside the Model URI field. Then, click “Next”.
```

1. We need to add the input connector EsbElement type to **InputConnectorEditPart** classes located inside **integrationstudio/gmf/esb/diagram/edit/parts/**. Make sure to replace the mediator name and corresponding number.

>     Example:    In "APIResourceInputConnectorEditPart.java" under "getMATypesForSource(IElementType relationshipType)"" method we need to add the following line,

Reviewer: ```suggestion
    Example:    You need to add the following line in the "APIResourceInputConnectorEditPart.java" file inside the "getMATypesForSource(IElementType relationshipType)" method.
```
